packages/yew/src/virtual_dom/renderable.rs

```diff
@@ -0,0 +1,205 @@
+use std::borrow::Cow;
+use std::rc::Rc;
+use std::sync::Arc;
+
+use super::{VChild, VList, VNode, VText};
+use crate::html::{ChildrenRenderer, IntoPropValue};
+use crate::{AttrValue, BaseComponent, Html};
+
+/// A trait implemented for types be rendered as a part of a Html.
+///
+/// Previously, a blanket implementation is given to `std::fmt::Display`
+/// and that is always converted to a `VText`.
+/// This trait allows types to define a virtual dom layout that itself should be rendered into via
+/// `html!`.
+pub trait Renderable {
```

Comment:
I'm not keen on the `Renderable` name. I would prefer it to be `Render`. That is more in line with how traits are named in Rust (e.g `Display`, not `Displayable`; `Hash`, not `Hashable`, etc)

Reply:
I changed this to `ToHtml`, which aligns with std traits `ToString` and `ToOwned`.